testnet/stacks-node/src/burnchains/mocknet_controller.rs

```diff
@@ -99,13 +101,22 @@ impl BurnchainController for MocknetController {
     fn get_stacks_epochs(&self) -> Vec<StacksEpoch> {
         match &self.config.burnchain.epochs {
             Some(epochs) => epochs.clone(),
-            None => vec![StacksEpoch {
-                epoch_id: StacksEpochId::Epoch20,
-                start_height: 0,
-                end_height: STACKS_EPOCH_MAX,
-                block_limit: ExecutionCost::max_value(),
-                network_epoch: PEER_VERSION_EPOCH_2_0,
-            }],
+            None => vec![
+                StacksEpoch {
+                    epoch_id: StacksEpochId::Epoch20,
+                    start_height: 0,
+                    end_height: 1,
+                    block_limit: ExecutionCost::max_value(),
+                    network_epoch: PEER_VERSION_EPOCH_2_0,
+                },
+                StacksEpoch {
+                    epoch_id: StacksEpochId::Epoch21,
+                    start_height: 1,
```

Comment:
Instead of hard coding start heights, could we read them from the config fil?
Thanks!

Reply:
Yup, seconded.

Reply:
Seems like this would create multiple ways to specify epoch start heights? How does this new config differ and interact with the existing one? e.g.
```toml
[[burnchain.epochs]]
epoch_name = "2.1"
start_height = 3
```

Reply:
That's what I was thinking. If we really need to customize the start height for epoch 2.1, we can use the existing way to configure epochs.

So should I keep the new configuration or revert it?

Reply:
Reverted.